src/app/course/add-course/add-course.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { AddCourseComponent } from "./add-course.component";
+import { RouterModule } from "@angular/router";
+import { SharedPipesModule } from "src/app/core/pipes/shared-pipes/shared-pipes.module";
```

Comment:
shared pipes should be placed in "app/shared/index
so your import should look like "../../shared/index